.github/workflows/release.yml

```diff
@@ -0,0 +1,74 @@
+name: Release RubyGems
+
+on:
+  push:
+    branches:
+      - release/*
+
+jobs:
+
+  pre-release-check:
+    name: Perform checks before releasing
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Check commit has been pushed on origin/main
```

Comment:
this seems really useful! I'd like to see this either factored out as a stand-alone action, or made part of the https://github.com/cucumber/action-create-github-release action so that we can't create releases from commits that are not on the main branch.

Reply:
This has to be done for each release workflow.
The action-create-github-release is not part of all release workflows with cucumber-expressions

A stand-alone action may fit

Reply:
I've added that to the checklist here: https://github.com/cucumber/common/issues/1688